tests/SelfTest/UsageTests/Matchers.tests.cpp

```diff
@@ -1079,6 +1079,7 @@ struct MatcherWasMovedOrCopied : std::exception {
 struct ThrowOnCopyOrMoveMatcher : Catch::Matchers::MatcherGenericBase {
     ThrowOnCopyOrMoveMatcher() = default;
 
+    // NOLINTBEGIN(readability-redundant-member-init)
```

Comment:
Same as the other lint here, is it still needed after a code cleanup?

Reply:
Same here.